src/Calculators/Dominion/SpellCalculator.php

```diff
@@ -67,6 +67,33 @@ public function canCast(Dominion $dominion, string $spell): bool
         );
     }
 
+    /**
+     * Returns whether spell $type for $dominion is on cooldown.
+     *
+     * @param Dominion $dominion
+     * @param string $spell
+     * @return bool
+     */
+    public function isOnCooldown(Dominion $dominion, string $spell): bool
+    {
+        $spellInfo = $this->spellHelper->getSpellInfo($spell, $dominion->race);
+
+        if (isset($spellInfo['cooldown'])) {
+            $spellLastCast = DB::table('dominion_history')
+                ->where('dominion_id', $dominion->id)
+                ->where('event', 'cast spell')
+                ->where('delta', 'like', '%'.$spell.'%')
```

Comment:
```suggestion
                ->where('delta', 'like', "%{$spell}%")
```

String interpolation for these kind of things for consistency across the rest of the codebase.

I don't really like doing a WHERE X LIKE %% on a JSON field tbh. Candidate for refactoring in the future.